### PR TITLE
Table relationships

### PR DESCRIPTION
**Pull request by KludgeKML**
Adds a report in: TableRelationshipReportGenerator

Relatively simple test - looks for "the table" in the text of the content item, and checks if an actual table exists in that content item. Reports only on invalid items.

### CODE REVIEW

File: src/utils/html_table_extractor.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import re
 from bs4 import BeautifulSoup
 
-
 class HtmlTableExtractor:
 
     no_th_css = "table > :not(thead):first-child > tr:first-child > :not(th)"
     no_row_header_css = "table tbody tr:nth-child(2) th:not([scope=row]), table tbody tr:first-child *:not(th):first-child"
     two_column_table_css = "td:first-child:nth-last-child(2), td:first-child:nth-last-child(2) ~ td"
+    table_mentions = "the table"
```
Comment (edwardkerry) [inline src/utils/html_table_extractor.py:9]: Might we also want to test for things like "see table" or "in table"?
I know there is no end to potentials and have missed conversations around this, so totally fine with it as it stands as well.

Comment (KludgeKML) [reply src/utils/html_table_extractor.py:9]: Hmm, maybe for the first in-earnest run I should just make it "table" but output the mentions in the report, see what that pulls up.

Comment (KludgeKML) [reply src/utils/html_table_extractor.py:9]: No, that's probably too much. But "see table" and "in table", yes. Good catch.

File: src/scripts/extract_items_from_content_store.python [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import sys
+
+# The purpose of this script is to take a list of URLs and extract the
+# preprocessed content items for those urls from the store into another csv file
+# that can be passed to the report runner. This makes it quicker to test new and
+# existing reports where you already know one URL with an issue they should
+# handle.
+
+gzipped_content_store_file_path = "downloads/preprocessed_content_store_210722.csv.gz"
+target_file_path = "downloads/abridged_content_store_210722.csv.gz"
```
Comment (edwardkerry) [inline src/scripts/extract_items_from_content_store.python:12]: Do we want to leave these hard-coded in here?
In fact could this script be it's own PR separate from the table relationship report?

**Review by edwardkerry — APPROVED** [review]
Looks good to me, one query about the commit for the script belonging in this PR but not blocking

File: src/scripts/extract_items_from_content_store.python (hunk 2 above)
Comment (edwardkerry) [reply src/scripts/extract_items_from_content_store.python:12]: If leaving hard-coded, perhaps call out they are examples and will need to be updated, but then I suppose that's also self explanatory

**Comment by KludgeKML** [comment]
Hmm, yeah, I was dithering about that. I think if you think it looks off I'll take it out. Ideally I should make that script a bit more script-y rather than hardcoded, but I think that's a job for if we decide this isn't basically a one-off run.
